repository/Seaside-Core.package/WASessionCookieProtectionFilter.class/class/browserSessionIDCookieKey.st

```diff
@@ -0,0 +1,4 @@
+private - constants
+browserSessionIDCookieKey
+
+	^ 'seaside_browser_session'
```

Comment:
Why is this on the class side?

Reply:
I move it to the instance side. It was used to have this key available from test methods in our codebase, but it makes no sense now.